data/scripts/actions/quests/children_of_the_revolution/mission4_levers_puzzle.lua

```diff
@@ -0,0 +1,86 @@
+local puzzle = {
+[1] = Position(33352, 31126, 5),
+[2] = Position(33353, 31126, 5),
+[3] = Position(33354, 31126, 5),
+[4] = Position(33355, 31126, 5)
+}
+
+local function revertLever(fromPosition)
+	if Tile(fromPosition):getItemById(9126) then
+		Tile(fromPosition):getItemById(9126):transform(9125)
+	end
+end
+
+local function puzzle1()
```

Comment:
We can redo this function in order to avoid repetitions, I'll make the script and send the suggestion here